lib/explorer/polars_backend/series.ex

```diff
@@ -379,11 +382,23 @@ defmodule Explorer.PolarsBackend.Series do
 
   # Helpers
 
-  defp scalar_rhs(scalar, lhs),
-    do:
-      scalar
-      |> List.duplicate(PolarsSeries.size(lhs))
-      |> Series.from_list()
+  defp apply_scalar_on_rhs(fun_name, %Series{} = left, scalar) when is_atom(fun_name) do
+    df =
+      DataFrame.mutate_with(DataFrame.new(left: left), fn ldf ->
```

Comment:
Oh, just one note. Make sure `DataFrame.new` is using the polars backend. Someone may have configured another backend globally. The same below.

Reply:
Fixed. Using the Polars backend to create it now. Thanks!